### PR TITLE
browser-ext/chrome: enable for shell scripts

### PR DESCRIPTION
**Pull request by mkovacs**
Enabling the Chrome extension for shell scripts.
Tested locally.

### CODE REVIEW

File: client/browser-ext/chrome/extension/annotations.js [hunk 1]
```diff
@@ -271,7 +271,7 @@ function addPopover(el) {
 				if (json.Data) {
 					const f = json.FmtStrings;
 					const doc = json.DocHTML ? `<div>${json.DocHTML.__html}</div>` : "";
-					html = `<div><div class=${styles.popoverTitle}>${f.DefKeyword || ""}${f.DefKeyword ? " " : ""}<b style="color:#4078C0">${f.Name.Unqualified}</b>${f.NameAndTypeSeparator || ""}${f.Type.ScopeQualified === f.DefKeyword ? "" : f.Type.ScopeQualified}</div>${doc}<div class=${styles.popoverRepo}>${json.Repo}</div></div>`;
+					html = `<div><div class=${styles.popoverTitle}>${f.DefKeyword || ""}${f.DefKeyword ? " " : ""}<b style="color:#4078C0">${f.Name.Unqualified}</b>${f.NameAndTypeSeparator || ""}${f.Type.ScopeQualified === f.DefKeyword ? "" : f.Type.ScopeQualified || ""}</div>${doc}<div class=${styles.popoverRepo}>${json.Repo}</div></div>`;
```
Comment (uforic) [inline client/browser-ext/chrome/extension/annotations.js]: What did the addtion of "" at the end of ```${f.Type.ScopeQualified === f.DefKeyword ? "" : f.Type.ScopeQualified || ""}``` protect against? Perhaps against f.Type.ScopeQualified being null?

Just want to make sure this is going to behave as expected.

File: client/browser-ext/chrome/extension/inject.js [hunk 2]
```diff
@@ -203,7 +203,7 @@ class InjectApp extends React.Component {
 		const pathParts = path.split("/");
 		let lang = pathParts[pathParts.length - 1].split(".")[1] || null;
 		lang = lang ? lang.toLowerCase() : null;
-		const supportedLang = lang === "go" || lang === "java";
+		const supportedLang = lang === "go" || lang === "java" || lang === "sh" || lang === "bash";
 		return window.location.href.split("/")[5] === "blob" && document.querySelector(".file") && supportedLang;
 	}
```
Comment (uforic) [inline client/browser-ext/chrome/extension/inject.js]: This LGTM

File: client/browser-ext/chrome/extension/annotations.js (hunk 1 above)
Comment (mkovacs) [inline client/browser-ext/chrome/extension/annotations.js]: It prevents showing "undefined" when there's no type information for the def.

Comment (uforic) [inline client/browser-ext/chrome/extension/annotations.js]: Ah, roger. Cool, LGTM.

**Pull request merged**